Documentation/Makefile

```diff
@@ -13,6 +13,7 @@ endif
 FOOTER		:= footer.txt
 SRC1		+= crit.txt
 SRC1		+= compel.txt
+SRC1		+= amdgpu_plugin.txt
```

Comment:
The name `amdgpu_plugin` (i.e., `man amdgpu_plugin`) is global.
What do you think about using something like `criu-amdgpu-plugin` instead?